chronograf.go

```diff
@@ -606,6 +606,14 @@ type User struct {
 	Scheme      string      `json:"scheme,omitempty"`
 }
 
+// UserQuery represents the attributes that a user may be retrieved by.
+// It is predominantly used in the UsersStore.Get method.
+type UserQuery struct {
```

Comment:
prob add `Scheme` to this?

Reply:
We could, but it feels premature to me. Happy to add it if you feel like we should.